.github/workflows/geoip.yml

```diff
@@ -22,3 +26,9 @@ jobs:
          ref: "refs/tags/monthly"
          replace: true
          files: IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP
+
+     - name: Publish release with current date
+       uses: cb80/pubrel@latest
```

Comment:
Relies on a private person repository that could change or stop being maintained.

Reply:
Yes, I used it because it is already in use:
https://github.com/OpenRA/GeoIP-Database/blob/b1c399bf028024cdde9a4e5fab940f14677bb784/.github/workflows/geoip.yml#L19-L20